helix-term/src/commands/typed.rs

```diff
@@ -1746,6 +1792,13 @@ pub const TYPABLE_COMMAND_LIST: &[TypableCommand] = &[
             fun: buffer_previous,
             completer: None,
         },
+        TypableCommand {
+            name: "rename",
+            aliases: &["r"],
```

Comment:
I'd prefer that `:r` be the alias for `:read` (https://github.com/helix-editor/helix/pull/3966). I'm not sure what the alias for `:rename` should be - `:mv` doesn't seem ideal; maybe `:rn`? Or is that too close to `:rm`?

Reply:
`:rnm` works for me 👍